CLAUDE.md

```diff
@@ -0,0 +1,110 @@
+# CLAUDE.md
+
+This file provides guidance to Claude Code (claude.ai/code) when working with code in this repository.
+
+## Essential Commands
+
+**Build and test:**
+
+```bash
+./gradlew build           # Build entire project
+./gradlew check           # Run all tests and linting
+
+# Note that each directory typically has a build.gradle file, but the available tasks follow similar conventions.
+
+# Java code.
+./gradlew spotlessApply   # Java code formatting
+
+# Python code.
+./gradlew :metadata-ingestion:testQuick     # Fast Python unit tests
+./gradlew :metadata-ingestion:testQuick     # Fast Python unit tests
+./gradlew :metadata-ingestion:lint          # Python linting (ruff, mypy)
```

Comment:
Probably also add `lintFix` command to auto-fix these?

Reply:
updated

Reply:
added